pkg/manager/impl/project_manager.go

```diff
@@ -70,9 +70,8 @@ func (m *ProjectManager) UpdateProject(ctx context.Context, projectUpdate admin.
 		return nil, err
 	}
 
-	// Run validation on the request, specifically checking for labels, and return err if validation does not succeed.
-	err = validation.ValidateProjectLabels(projectUpdate)
-	if err != nil {
+	// Run validation on the request and return err if validation does not succeed.
+	if err := validation.ValidateProject(projectUpdate); err != nil {
```

Comment:
This breaks archiving/activating project feature unfortunately because in that case, project.Name and project.Description is not required.

Reply:
I think for any update operation, it is reasonable to allow certain fields to be unset.

Reply:
+1, we should only do a merge update in that case and not override empty fields